pkg/core/state/transfer_log.go

```diff
@@ -0,0 +1,49 @@
+package state
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/io"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+// TransferSize is a size of a marshaled Transfer struct in bytes.
+const TransferSize = util.Uint160Size*2 + 8 + 4 + 4 + util.Uint256Size
```

Comment:
`util.Uint256Size * 2` for both `AssetID` and `Tx`. Though we can probably save on `AssetID` as we only have two assets to track.

Reply:
Fixed.